app.js

```diff
@@ -0,0 +1,65 @@
+// Start of Home page code
+const statusContainer = document.getElementById('status-container');
+if (statusContainer !== null) {
+    // if in Home page and document is loaded
+    document.addEventListener('DOMContentLoaded', updateStatus);
+
+    // Call UpdateCountDown function every 1 second
+    setInterval(UpdateCountDown, 1000);
+}
+
+// Start of count down code
+const seconds = 10;
+let timer = seconds;
+
+timer = seconds;
+function UpdateCountDown() {
+    const countDown = document.getElementById('countDown');
+    countDown.innerText = `Time to check: ${timer.toString()}`;
+    timer--;
+
+    if (timer === 0) {
+        updateStatus();
+        timer = seconds;
+    }
+}
+// End of count down code
+
+// update the label & background of the center status in Home page
+function updateStatus() {
+    // const statusContainer = document.getElementById('status-container');
+
+    // Get the center data from Local Storage
+    const center = JSON.parse(localStorage.getItem('center'));
+
+    // If center marked as full
+    if (center !== null && center.isFull !== undefined && center.isFull) {
+        statusContainer.className = 'full-status-container';
+        statusContainer.innerText = 'Center is full';
+    } else {
+        statusContainer.className = 'default-status-container';
+        statusContainer.innerText = 'Still available'
+    }
+}
+// End of Home page code
+
+// Start of Admin page code
+const btnMarkFull = document.getElementById('full-button');
+const btnMarkVacant = document.getElementById('vacant-button');
+
+if (btnMarkFull !== null) {
+    btnMarkFull.addEventListener('click', e => this.changeStatus(e));
+    btnMarkVacant.addEventListener('click', e => this.changeStatus(e));
+}
+
+function changeStatus(e) {
+    const center = {
+        isFull: false
+    }
+
+    if (e.target === btnMarkFull) {
+        center.isFull = true;
+    }
+    localStorage.setItem('center', JSON.stringify(center));
+}
```

Comment:
I believe readability will be much improved if this function was split into two functions, one for each case, and each one was only called by the respective button.

Reply:
It seems like you missed my point regarding this comment. This is optional but here is a sample that fulfills this request:
```javascript
btnMarkFull.addEventListener('click', setFull);
btnMarkVacant.addEventListener('click', setVacant);

function setFull() {
    localStorage.setItem('center', JSON.stringify({isFull: true}))
}

function setVacant() {
    localStorage.setItem('center', JSON.stringify({isFull: false}))
}
```